CDCgov/prime-reportstream
# Experience-7945: Update additional validation error messages

This changeset updates the human-readable descriptions matching the error codes we get from the validator.  It requires a bit more logic regarding the markup so the descriptions have been split out into a separate ValidationErrorMessage component rather than being a simple dictionary of `<error_code, description>`.

Test Steps:
1. Log in as admin
2. Go to /file-handler/validate
3. Upload a file with erroneous/missing data as discussed in [this proposal](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/proposals/0017-serializer-error-codes.md)
    - Alternatively, you can also mock the responses such that a given error code is returned
5. Verify that the correct error descriptions are shown 

## Changes
![image](https://user-images.githubusercontent.com/2421042/221595768-1dbea8bc-64eb-493d-aae3-d7e1f860962e.png)
![image](https://user-images.githubusercontent.com/2421042/221595740-d4abfb57-416a-4b13-ba97-51e87b9acc8c.png)

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #7945
